sklearn/manifold/_spectral_embedding.py

```diff
@@ -339,7 +355,13 @@ def spectral_embedding(
         X = random_state.randn(laplacian.shape[0], n_components + 1)
         X[:, 0] = dd.ravel()
         X = X.astype(laplacian.dtype)
-        _, diffusion_map = lobpcg(laplacian, X, M=M, tol=1.0e-5, largest=False)
+        # Until scikit-learn minimum scipy dependency <1.4.0 we require high
+        # tolerance as explained in:
+        # https://github.com/scikit-learn/scikit-learn/pull/13707#discussion_r314028509
+        tol = max(1e-5, 1e-5 if eigen_tol is None else eigen_tol)
```

Comment:
May we update the docstrings to explain this clipping behavior for `eigen_tol` + lobpcg`.

Reply:
From an irl discussion with @ogrisel and @glemaitre, we don't think clipping is a good solution here. We should let the user chose the tol he wants. Instead we can document in the parameter description that it's advised to no use a too low tolerance when scipy < 1.4